OasisDEX/oasis-borrow
# Add mixpanel tracking to stETH Earn flow

# [Add mixpanel tracking to stETH Earn flow](https://app.shortcut.com/oazo-apps/story/6165/add-mixpanel-tracking-to-steth-earn-flow)

  
## Changes 👷‍♀️
 - added new events to corresponding user actions regarding AAVE steth/eth flow
  
## How to test 🧪
 - [DEV] perform various actions on steth flow, you should see events in the console for example:
 ```
 Mixpanel Event:  btn-click {
  id: 'ConfirmRisk',
  page: 'OpenEarnSTETH',
  section: 'OpenPosition',
  distinct_id: 'test_id',
  '$current_url': 'http://localhost:3000/#multiply',
  '$initial_referrer': '$direct',
  '$initial_referring_domain': '$direct',
  '$user_id': 'test_user_id'
}
```
 - [PROD/STAGING] if you have access to the mixpanel you should see new events in `OpenEarnSTETH` and `ManageSTETH` pages
